js/src/dashboard/summary-section/index.js

```diff
@@ -106,17 +107,21 @@ const PaidPerformanceCard = () => {
 };
 
 export default function SummarySection() {
-	const { adsSetupComplete } = glaData;
+	const { loaded, data: adsCampaignsData } = useAdsCampaigns();
+	if ( ! loaded ) {
+		return null;
+	}
+	const showCampaignPromotionCard = Boolean( ! adsCampaignsData?.length );
```

Comment:
The `!` operator already does the boolean typecasting. The second time boolean typecasting `Boolean()` is redundant.